forte_wrapper/elastic/elastic_search_processor.py

```diff
@@ -53,18 +53,21 @@ def default_configs(cls) -> Dict[str, Any]:
             datapack.
             - response_pack_name_prefix: the pack name prefix to be used
             in response datapacks.
-            - only_pass_content: boolean, defines if only the content field
-            of the datapack should be passed to the multipack, default
-            is True. If set False, the original datapack will be passed
-            to the multipack.
+            - indexed_text_only: boolean, defines whether the returned
+            value from the field (as specified by the field configuration)
+             will be considered as plain text. If True, a new data pack
+             will be created and the value will be used as the text for
```

Comment:
The indent here seems to be off (this may cause a problem when rendering documentation with sphinx)